Automattic/wp-calypso
# Display the media export URL in the export page.

## Summary
This PR makes the export page and the underlying `exporter` substate tree be aware of the newly-added media export URL and data shape changes from the response of the exporting endpoint. After a successful export, it shows a notification like this:
![image](https://user-images.githubusercontent.com/1842898/46328814-694b8e80-c5d7-11e8-9067-f1a698f417b3.png)

It's part of the GM project of supporting media library export. For more details, please refer to p2-p6OVho-dR.

## Testing instructions
1. Apply code-D18988
1. Go to http://calypso.localhost:3000/settings/export/
1. Click "Export All"
1. The "Download" button should link to the WXRs archive URL as always.
1. The "Download Media" button should link to the media library archive URL.
